client/src/test/suite/extension.test.ts

```diff
@@ -306,4 +323,42 @@ suite('Extension Test Suite', function (this: Suite) {
     populateXsdSchemaFiles(TEST_OUT_DIR)
     assert(!fs.existsSync(testXsdPath))
   })
+  test('getRepo returns repository', async () => {
+    const repo = pushContent.getRepo()
+    assert.notStrictEqual(repo.rootUri, undefined)
+  })
+  test('push with no conflict', async () => {
+    stubRepo.commit = sinon.stub().throws()
+    stubRepo.pull = sinon.stub().returns(Promise.resolve())
+    stubRepo.push = sinon.stub().returns(Promise.resolve())
+    await pushContent._pushContent(getRepo, reporter('Successful content push.'), ignore)
```

Comment:
`pushContent.__pushContent(...)` returns an async function, not a promise, awaiting it doesn't do anything. You will need to call this returned function for your assertion to run.